de.fhdo.lemma.service.open_api/src/de/fhdo/lemma/service/open_api/exceptions/ComplexTypeException.xtend

```diff
@@ -0,0 +1,9 @@
+package de.fhdo.lemma.service.open_api.exceptions
+
+import java.lang.Exception
+
+class ComplexTypeException extends Exception {
```

Comment:
Exceptions should have descriptive names, e.g., `ComplexTypeCastException` or `InvalidComplexTypeException`. Also: Comment